From 3ce1906044173503ef21528337a04c96a9cf24ef Mon Sep 17 00:00:00 2001
From: Christophe Roullier <christophe.roullier@foss.st.com>
Date: Mon, 11 Mar 2024 14:32:20 +0100
Subject: [PATCH] arm64: dts: st: enable FDCAN1/FDCAN3 on stm32mp257f-ev1 board

Enable support of FDCAN1 and FDCAN3

Signed-off-by: Christophe Roullier <christophe.roullier@foss.st.com>
Change-Id: I140336f53ed91bb14048dbd9e4a769eb03fbb0af
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/365906
Reviewed-by: Gwenael TREUVEUR <gwenael.treuveur@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
index fd8075c0cf43..a2533e52754b 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -488,14 +488,14 @@ &m_can1 {
 	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&m_can1_pins_a>;
 	pinctrl-1 = <&m_can1_sleep_pins_a>;
-	status = "disabled";
+	status = "okay";
 };
 
 &m_can3 {
 	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&m_can3_pins_a>;
 	pinctrl-1 = <&m_can3_sleep_pins_a>;
-	status = "disabled";
+	status = "okay";
 };
 
 &mlahb {
-- 
2.39.5

